src/client/driver/command-executors/execute-action-command.js

```diff
@@ -66,6 +67,8 @@ function runAutomation (element, command) {
         automation = new RClickAutomation(element, command.options);
     else if (command.type === COMMAND_TYPE.doubleClick)
         automation = new DblClickAutomation(element, command.options);
+    else if (command.type === COMMAND_TYPE.hover)
+        automation = new HoverAutomation(element, command.options);
 
```

Comment:
There is so many `else if`. What's about creating an object  with `COMMAND_TYPE`'s as keys like we have for [error templates](https://github.com/DevExpress/testcafe/blob/master/src/errors/test-run/templates.js)?